mir-module/src/main/resources/META-INF/resources/hbs/derivate-fileList.hbs

```diff
@@ -41,6 +41,8 @@
                 <a href="{{href}}">
                   {{name}}
                 </a>
+                <!-- Show md5 sum -->
+                <div class="md5 hidden">{{getI18n "mir.derivate.file.MD5"}} {{md5}}</div>
```

Comment:
If `{{getI18n "mir.derivate.file.MD5"}}` is wrapped in a `<span class="file_md5_label"/>` element other applications are able to hide that part of the content.